common/src/main/java/com/hedera/block/common/constants/StringConstants.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.block.common.constants;
+
+import static com.hedera.block.common.constants.ErrorMessageConstants.CREATING_INSTANCES_NOT_SUPPORTED;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/** A class that hold common {@link String} literals used across projects. */
```

Comment:
also remove the '@link String` and improve the doc.

Reply:
agree

Reply:
removed